package.xml

```diff
@@ -21,14 +21,15 @@
   <!-- ROS dependencies. -->
   <depend>linear_feedback_controller_msgs</depend>
   <depend>rospy</depend>
+  <depend>xacro</depend>
 
   <!-- Python dpendencies. -->
-  <exec_depend>python-matplotlib</exec_depend>
-  <exec_depend>python-numpy</exec_depend>
+  <exec_depend>python3-matplotlib</exec_depend>
+  <exec_depend>python3-numpy</exec_depend>
 
   <!-- LAAS dependencies. -->
   <exec_depend>pinocchio</exec_depend>
-  <exec_depend>croccodyl</exec_depend>
+  <exec_depend>crocoddyl</exec_depend>
   <exec_depend>example-robot-data</exec_depend>
```

Comment:
Neither of those two are available as binaries. It should be documented that user has to download them manually